app/src/main/java/com/helio/app/model/MotionSensor.java

```diff
@@ -0,0 +1,25 @@
+package com.helio.app.model;
+
+import java.util.List;
+
+public class MotionSensor extends Sensor {
+    private String motionSensitivity;
```

Comment:
is this the same as `durationSensitivity` in helio-hub's `api.yml`? If this is a different attribute, would a `double` or `int` be more suitable?

Reply:
I changed it to int, but it is a "hh:mm" string on [APIspec](https://github.com/SDP-group22/helio-hub/blob/main/api-spec.yml). We should probably change the API to int, it is easier.

Reply:
https://github.com/SDP-group22/helio-hub/pull/4